lib/modules/apostrophe-workflow-modified-documents/index.js

```diff
@@ -115,5 +115,16 @@ module.exports = {
         group.items.push(self.__meta.name);
       }
     };
+    // Since this is an overview of many doc types, all that is required
```

Comment:
This is duplicated in #258

Reply:
That branch has been merged and there are no conflicts, so resolving.